roles/st2web/meta/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+galaxy_info:
+  description: Install st2web
+  author: humblearner
+  company: StackStorm
+  license: Apache
+  min_ansible_version: 2.2
+  tags: st2web
+  platforms:
+    - name: Ubuntu
+      versions:
+        - trusty
+        - xenial
+    - name: EL
+      versions:
+        - 6
+        - 7
+  categories:
+    - system
+dependencies:
```

Comment:
So did you check this behavior?

If you remove `nginx` role from the `stackstorm.yml`, will `st2web` run `nginx` as a dependency?
Would like to be sure here.